sim/mage/item_sets_pve.go

```diff
@@ -150,7 +150,7 @@ var ItemSetArcanistInsight = core.NewItemSet(core.ItemSet{
 			mage.RegisterAura(core.Aura{
 				ActionID: core.ActionID{SpellID: 456402},
 				Label:    "S03 - Item - T1 - Mage - Damage 6P Bonus",
-				OnInit: func(aura *core.Aura, sim *core.Simulation) {
+				OnGain: func(aura *core.Aura, sim *core.Simulation) {
```

Comment:
Just curious why this can't be done in the OnInit? The FoF proc chance doesn't change dynamically during an encounter

Reply:
Because OnInit only happens _once_ before the first iteration, so doing it like that the bonus is only active for the first iteration. Finally got to setup a dev VM for this again and want to get the concurrency port merged, and all of those showed up in the test there :D